charts/kube-vip/values.yaml

```diff
@@ -59,12 +59,12 @@ resources: { }
 #   cpu: 100m
 #   memory: 128Mi
 
-nodeSelector:
-  node-role.kubernetes.io/master: "true"
+nodeSelector: {}
+  # node-role.kubernetes.io/controlplane: "true"
 
 tolerations:
   - effect: NoSchedule
-    key: node-role.kubernetes.io/master
+    key: node-role.kubernetes.io/controlplane
```

Comment:
Why do we change from `master` to `controlplane`?

Reply:
As mentioned in the commit, upstream Kubernetes has switched it's naming over to controlplane, for _reasons_.
This can also been seen anywhere in their docs, and new clusters are by default using the controlplane naming.

Reply:
@yaocw2020 are we OK merging this?

Reply:
According to the Kubernetes source code, the label key is `node-role.kubernetes.io/control-plane`, not `node-role.kubernetes.io/controlplane`.

https://github.com/kubernetes/kubernetes/blob/2e865cef76d9623dee07f464bd87a16ef7abefc2/cmd/kubeadm/app/constants/constants.go#L237

Reply:
```
dan@k8s01:~$ kubectl describe node | grep node-role.kubernetes.io/control-plane
                    node-role.kubernetes.io/control-plane=
                    node-role.kubernetes.io/control-plane=
                    node-role.kubernetes.io/control-plane=
```

Good catch!